tests/bootstrap.php

```diff
@@ -1,8 +1,12 @@
 <?php
 
 if (!defined('APPLICATION_ROOT_DIR')) {
-    define('APPLICATION_ROOT_DIR', __DIR__ . DIRECTORY_SEPARATOR . 'app');
+    define('APPLICATION_ROOT_DIR', __DIR__ . DIRECTORY_SEPARATOR . 'app/');
```

Comment:
I think they dont have a slash, at least we would need to fix up some of the core internals as there is another slash appended usually..